pkg/apis/jaegertracing/v1/jaeger_types.go

```diff
@@ -399,6 +399,9 @@ type JaegerCassandraCreateSchemaSpec struct {
 	// +optional
 	Mode string `json:"mode,omitempty"`
 
+	// +optional
+	Timeout *int64 `json:"timeout,omitempty"`
```

Comment:
Perhaps leave a comment that this is in line with what is being used downstream? Intuitively, I would expect a "duration", similar to what we have elsewhere here (see `ReadTTL`), but looking at the cronjob spec, I see that it accepts an integer representing the number of "seconds".

Reply:
I was wondering how to handle duration parsing errors? I decided to go with using a default timeout (120s), but maybe blocking job deployment would make more sense? I couldn't find out how to test such behavior though (that's why I decided to use the default timeout).

Reply:
In the operator, we typically fall back to a sensible default + log at info/warn level whenever possible. For instance, we fallback to "in memory" storage when the storage type is set to an unknown type.